docs/release/trg-1/trg-1-08.md

```diff
@@ -0,0 +1,27 @@
+---
+title: TRG 1.08 - Interface documentation (APIs)
+sidebar_position: 1
+---
+
+| Status | Created     | Post-History                                          |
+|--------|-------------|-------------------------------------------------------|
+| Active | 01-Aug-2024 |                                                       |
+| Draft  | 21-Mar-2024 | Transferred from Release Acceptance Criteria into TRG |
+
+## Why
+
+Interface documentation is crucial to interact with components. It fundamentally enables others to consume your APIs.
+
+## Description
+
+- The interface documentation **must** be stored in the folder `/docs/api` as `openAPI.yaml`
```

Comment:
I would change this sentence, to not force anybody to keep the openapi file in the repository, from our perspective it is an artifact so it's generated and published on every release  (currently on swaggerhub, in the future likely on github pages).
 
As long as it is reachable from a link available in the metadata file, it should be ok, so every project can then decide where to store it.

Reply:
@ndr-brt i adjusted it. if its roughly okay, pls resolve and finetune it in another pr if necessary.